### DISCUSSION TITLE
no "Open Completions Panel" option

### DISCUSSION THREAD

**Discussion by serkul** [skills/.github#275] [Q&A]
`Open Completions Panel` doesn't appear, I only have `Accept line` and `Always show toolbar` after clicking `...`. I have tried it locally in VS code and the result is the same. Is this feature no longer available with Copilot or I'm missing something?

**Comment by serkul** [comment]
Seems like it is not in the GUI, but I can still synthesise  suggestions with a short cut, (`control + enter` by default)

**Reply by Carewj00** [reply]
solution worked for me

**Reply by AswinSuku** [reply]
hey do you purchased free trial of copilot ($10/month or $100/year after trial period) coz i haven't purchased it so i think that it is not giving me suggestions because of that. : (

**Reply by serkul** [reply]
I have the subscription

**Comment by durume** [comment]
@serkul I had the same issue. In my case, after installing ESLint and press 'ctrl + Enter' with little waiting, the 'Open Completion Panel' works fine.
